packages/back-end/src/app.ts

```diff
@@ -547,6 +547,13 @@ app.post("/file/upload/:filetype", discussionsController.postImageUploadUrl);
 app.get("/admin/organizations", adminController.getOrganizations);
 app.post("/admin/organization/:id/populate", adminController.addSampleData);
 
+// Meta info
+app.get("/meta/ai", (req, res) => {
+  res.json({
+    enabled: !process.env.OPENAI_API_KEY,
```

Comment:
Is this a typo?  Should it be `!!` instead?

Reply:
Almost certainly a typo though not sure how I missed it. Going to do some testing after updating

Reply:
Tested; this typo was likely a leftover from prior testing